python/mxnet/contrib/onnx/_import/op_translations.py

```diff
@@ -214,9 +214,25 @@ def conv(attrs, inputs, cls):
     new_attrs = translation_utils._fix_bias('Convolution', new_attrs, len(inputs))
 
     new_attrs = translation_utils._fix_channels('Convolution', new_attrs, inputs, cls)
-
-    return 'Convolution', new_attrs, inputs
-
+    kernel = new_attrs['kernel']
+    stride = new_attrs['stride'] if 'stride' in new_attrs else []
+    padding = new_attrs['pad'] if 'pad' in new_attrs else []
+    dilations = new_attrs['dilate'] if 'dilate' in new_attrs else []
+    num_filter = new_attrs['num_filter']
+    num_group = new_attrs['num_group']
+    no_bias = new_attrs['no_bias'] if 'no_bias' in new_attrs else 0
```

Comment:
should we use Flase instead of 0 here?

Reply:
changed. With regards to unit tests, for this whole module we are using ONNX's backend test framework - https://github.com/onnx/onnx/blob/master/docs/OnnxBackendTest.md. It runs tests for each operator's validity as well as tests for popular models.
These are the tests we are running for 'Pad' operator-
https://github.com/apache/incubator-mxnet/blob/master/tests/python-pytest/onnx/onnx_backend_test.py#L65-L67
And these are the tests we are running for convolution -
https://github.com/apache/incubator-mxnet/blob/master/tests/python-pytest/onnx/onnx_backend_test.py#L74-L75

Reply:
can test_basic_conv_with_padding be added ?